src/app/components/client/ExposureCard.tsx

```diff
@@ -67,10 +70,14 @@ export const ExposureCard = (props: ExposureCardProps) => {
     exposureData,
     exposureDetailsLink,
     statusPillType,
+    locale,
   } = props;
 
   const l10n = useL10n();
   const [exposureCardExpanded, setExposureCardExpanded] = useState(false);
+  const dateFormatter = new Intl.DateTimeFormat(locale, {
+    dateStyle: "medium",
+  });
```

Comment:
suggestion: Add a comment about what `dateStyle: "medium` means? (Example format, maybe?)

Reply:
Added a link to the docs in 51c9261f0, though technically how it gets formatted depends on `locale` _and_ the user's browser. But for `en`, it's basically short-but-not-ambiguous, in the format "Jul 10, 2023".